packages/bsp/common/usr/lib/armbian/armbian-ramlog

```diff
@@ -112,6 +112,14 @@ case "$1" in
 	write)
 		syncToDisk
 		;;
+	postrotate)
+		cd /var/log.hdd/
+		find . -type f -print | grep -E -v "(\.gz|\.xz|\.[0-9]|armbian-ramlog)" | while IFS= read -r  file
+		do
+	    		dest="/var/log/$file"
```

Comment:
Good idea, but since the Pull request has been already pulled, it is no longer useful to modify it in this fork. Let us hope one of the developers will see your message.